book/content/reproducibility/reproducibility.md

```diff
@@ -4,7 +4,8 @@
 No previous knowledge needed.
 
 ## Summary
-There are several definitions of reproducibility in use.
+There are several definitions of reproducibility in use, and these can be found in [2.3 Definitions](https://github.com/RostiReadioff/the-turing-way/blob/master/book/content/reproducibility/03/definitions.md).
```

Comment:
Thanks so much for adding this @RostiReadioff.

This link goes to _your_ repository but actually it will need to be a *relative* link to the definitions.md file rather than the absolute link.

I've slightly re-worded the sentence because the numbering might move around as we move sections about and so hopefully this wording is a little more resilient to those changes in other parts of the book.

I think the following suggestion should work. What do you think?

```suggestion
There are several definitions of reproducibility in use, and we discuss thes in more detail in in the [definitions](../03/definitions.md) section of this chapter.
```

Reply:
Good! There is a typo in your suggestion.

Reply:
I have added in your suggestions 😊